accounts-management/manager.go

```diff
@@ -234,3 +236,27 @@ func (m *AccountsManager) GetVerifiedWalletAccount(address cryptotypes.Address,
 
 	return account, nil
 }
+
+func (m *AccountsManager) VerifySelectedChatAccountPassword(password string) (bool, error) {
+	m.mu.RLock()
+	defer m.mu.RUnlock()
+
+	if m.persistence == nil {
+		return false, ErrPersistenceMissing
+	}
+
+	profileKeypair, err := m.persistence.GetProfileKeypair()
+	if err != nil {
+		return false, errorspkg.Wrap(err, "error getting profile keypair")
+	}
+
+	if profileKeypair.MigratedToKeycard() {
+		return false, ErrKeypairIsNotKeycard
```

Comment:
We should add a new error:
```
ErrProfileMigratedToKeycard = errors.NewError(ErrCodeProfileMigratedToKeycard, "profile keypair is not a regular keypair, but keycard keypair", getErrorCategory)
```
In this context, if the profile keypair is migrated to a keycard it's an error checking for the password validation this way, we should return:
```suggestion
		return false, ErrProfileMigratedToKeycard
```

Reply:
Completely removed the function now